src/tools/spake2p/spake2p.cpp

```diff
@@ -66,7 +66,7 @@ int main(int argc, char * argv[])
 {
     bool res = false;
 
-    chip::Platform::MemoryInit();
+    TEMPORARY_RETURN_IGNORED chip::Platform::MemoryInit();
```

Comment:
I don't think that we should go any farther in case of memory initialization failure.

```suggestion
    SuccessOrDie(chip::Platform::MemoryInit());
```